ray-operator/controllers/ray/common/pod.go

```diff
@@ -189,6 +189,18 @@ func DefaultWorkerPodTemplate(instance rayiov1alpha1.RayCluster, workerSpec rayi
 		log.Info("Setting pod namespaces", "namespace", instance.Namespace)
 	}
 
+	// The Ray worker should only start once the GCS server is ready.
+	initContainer := v1.Container{
+		Name:            "wait-gcs-ready",
+		Image:           podTemplate.Spec.Containers[0].Image,
```

Comment:
It may be preferable to utilize [getRayContainerIndex](https://github.com/ray-project/kuberay/blob/19ddf045118f5618ca7c433f59743d3abfa44140/ray-operator/controllers/ray/common/pod.go#L466) to obtain the index.

Reply:
Updated.